src/js/pannellum.js

```diff
@@ -478,6 +478,12 @@ function onDocumentMouseMove(event) {
         var pitch = ((Math.atan(event.clientY / renderer.canvas.height * 2 - 1) - Math.atan(onPointerDownPointerY / renderer.canvas.height * 2 - 1)) * 180 / Math.PI * vfov / 90) + onPointerDownPitch;
         pitchSpeed = (pitch - config.pitch) * 0.2;
         config.pitch = pitch;
+
+        var verbose = config.verbose || false;
+        if(verbose){
+            console.log("Yaw = ", yaw*-1);
```

Comment:
That would be a bug in the hot spot rendering. I just fixed it, which unfortunately means the sign needs to be switched for the yaw of every hot spot in existing configuration files. It's rather embarrassing that I never noticed this.